tests/test_dse.py

```diff
@@ -373,19 +374,20 @@ def test_contracted_shape_after_blocking(self):
         assert a.halo == ((1, 1), (1, 1))
         assert Add(*a.symbolic_shape[0].args) == y0_blk_size + 2
         assert Add(*a.symbolic_shape[1].args) == z_size + 2
+
         # Check numerical output
         op0(time_M=1)
         exp = np.copy(u.data[:])
-        u.data_with_halo[:] = 0.
+        u.data_with_halo[:] = 0.5
```

Comment:
why?

Reply:
to minimize amount of computation on zeros